src/components/section/active-section/index.tsx

```diff
@@ -0,0 +1,24 @@
+import classNames from './active-section.module.scss';
+import Active from '../../challenges/active';
+import Constants from '../../constants/display-sections.js';
```

Comment:
I don't see an export statement in the constants file

Reply:
Also, prefer `import { CHALLENGES_SCREEN } from '../../constants/display-sections.js';`

Reply:
Done,
Ref: https://github.com/Real-Dev-Squad/website-roadmap/pull/8/commits/c5297db1ff9e6f0180a2600c4e7adb9ecad8f7a6